infrastructure/subscription_level/main.bicep

```diff
@@ -0,0 +1,14 @@
+param location string = 'uksouth'
+param resourceGroupName string = 'Digital-Feature'
```

Comment:
can we name it `rg-dtfs-dev-001`? something like that? otherwise we have to rename it again later on

Reply:
My current plan is to replicate the current environments as accurately as possible, including naming, then update them when I'm confident they correspond. (Perhaps I'm just making more work for myself though!)

Obviously I'm not going to get confused with the resource group though!

Note that once deployed, we can't rename things! I'm not sure if it is worth redeploying resources in dev/test/staging/prod just to have consistent naming.

Some bloke here has found a bit of a hacky workaround: https://azureis.fun/posts/Adding-a-friendly-name-to-Azure-Resources/

Reply:
We need to follow Azure's best practices in terms of naming conventions -> [https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-naming ).
So yes, we need to rename all resources

Reply:
We need to form a plan for renaming of production resources - Azure's weird decision to make name and id connected means a rename is a delete and create...
I'll look into that at the appropriate time.

If it's ok, I'll stick with my plan to follow best naming practice after I've produced a working deployment. (i.e. in a separate card near the end.)

Reply:
Ok cool

Reply:
I've raised https://ukef-dtfs.atlassian.net/browse/FN-462 to make this more explicit.